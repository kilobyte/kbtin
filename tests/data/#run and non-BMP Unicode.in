#hook close {#showme FIN;#end}
#run a echo abc𠀧𠀀𠀁𠀂
