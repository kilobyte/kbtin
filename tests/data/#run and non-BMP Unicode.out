abc𠀧𠀀𠀁𠀂
FIN
